samples/besu/simpleasset/migrations/2_deploy_contracts.js

```diff
@@ -1,9 +1,16 @@
 const InteroperationBaseClassERC20 = artifacts.require('InteroperationBaseClassERC20')
 const AliceERC20 = artifacts.require('./AliceERC20.sol')
 const BobERC20 = artifacts.require('./BobERC20.sol')
+const AliceERC721 = artifacts.require('./AliceERC721.sol')
+const InteroperationBaseClassAny = artifacts.require('InteroperationBaseClassERCAny')
+const AliceERC1155 = artifacts.require('./AliceERC1155.sol')
+
 
 module.exports = function (deployer) {
 	deployer.deploy(InteroperationBaseClassERC20)
```

Comment:
Should we deploy this contract if we are deploying `InteroperationBaseClassAny` later?
I think only one of them should be deployed at a time, may be have an `env` var to decide which one to deploy.